packages/vant/src/lazyload/README.md

```diff
@@ -73,7 +73,7 @@ app.use(Lazyload, {
 | --- | --- | --- | --- |
 | loading | Src of the image while loading | _string_ | - |
 | error | Src of the image upon load fail | _string_ | - |
-| preload | Proportion of pre-loading height | _string_ | - |
+| preload | Proportion of pre-loading height | _number_ | - |
```

Comment:
Thanks! Should the option name be `preLoad`?

```suggestion
| preLoad | Proportion of pre-loading height | _number_ | - |
```

Reply:
wait...